documentation/README.md

```diff
@@ -0,0 +1,72 @@
+# Documentation
+
+The documentation is built using [Docusaurus 2](https://docusaurus.io/). The deployment is done through a centralized build from [IOTA WIKI](https://github.com/iota-community/iota-wiki). To run a local instance the [IOTA WIKI CLI](https://github.com/iota-community/iota-wiki-cli) is used.
+
+## Prerequisites
+
+- [Node.js v14.14+](https://nodejs.org/en/)
+- [yarn](https://yarnpkg.com/getting-started/install)
+
+## Installation
+
+```console
+yarn
+```
+
+This command installs all needed dependencies.
+
+## Local Development
+
+```console
+yarn start
+```
+
+This command starts a local, wiki themed, development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+## Including .md file
+
+```console
+{@import <file path>}
+```
+
+Example:
+
+```console
+{@import ../../../../bindings/wasm/docs/api-reference.md}
+```
+
+## Default Structure
+
+IOTA projects usually house their Wiki documentation within the documentation folder of any project. This
+template provides a set structure to separate concerns based on the [Diataxis framework](https://diataxis.fr/), with
+some minor modifications.
+
+You do not need to use all of these sections, so please feel free to delete any which do not suit your needs.
+
+### Getting Started
+
+The [Getting Started folder](documentation/getting_started/README.md) should be the first section in your documentation. It should give the user a high level overview of the project, required prior knowledge, prerequisites, and ideally a quick set up guide, or "hello world".
+
+### Key Concepts
```

Comment:
Do we also want to keep It in this order?
Is key concepts abstracted enough? If it is more theory we should move it down in the sidebar as it won't be read by many people 🤔

Reply:
Maybe the  `Examples` & `Tutorials` need to move to the top?